pipenv/cli/options.py

```diff
@@ -182,6 +182,15 @@ def selective_upgrade_option(f):
     def callback(ctx, param, value):
         state = ctx.ensure_object(State)
         state.installstate.selective_upgrade = value
+        if value:
+            click.secho(
+                "The flag --selective-upgrade has been deprecated for removal."
+                "The flag is buggy and leads to inconsistent lock files.  "
```

Comment:
This will result in text like `deprecated for removal.The flag is buggy`

```suggestion
                "The flag --selective-upgrade has been deprecated for removal.  "
                "The flag is buggy and leads to inconsistent lock files.  "
```